src/lib/ng-package/package.transform.ts

```diff
@@ -134,9 +139,11 @@ const watchTransformFactory = (
       return createFileWatch(data.src, [data.dest]).pipe(
         tap(fileChange => {
           const { filePath, event } = fileChange;
-          const { sourcesFileCache } = cache;
+          const { sourcesFileCache, ngccProcessingCache } = cache;
           const cachedSourceFile = sourcesFileCache.get(filePath);
 
+          ngccProcessingCache.clear();
```

Comment:
Do we need to clear the cache on each file file? We don't watch node_modules therefore I assume that we shouldn't do this.

Reply:
I wanted to avoid a situation where `node_modules` was somehow affected between watch builds, as in that case the cache would incorrectly cause ngcc not to run. Even if we don't watch `node_modules` it's still possible for someone to change a file that is watched after something in `node_modules` has changed.

Reply:
I think this will not work because of the "local" processed files cache `processedModules`

Reply:
`NgccProcessor` is recreated in the transform pipeline, so it's not a persistent cache per entry-point.